ci-scripts/deploy-ccv2.sh

```diff
@@ -42,49 +44,99 @@ else
+echo "update server.ts for b2c storefront"
+
+sed -i "s%dist/storefrontapp%dist/$B2C_STORE/browser%gi" projects/storefrontapp/server.ts
+
+echo "---------------------------------------------------------------------------------------------------------------------------"
+echo "Verify server.ts has been updated for b2c dist"
+
+cat projects/storefrontapp/server.ts
+
+if grep -Fq "const distFolder = join(process.cwd(), 'dist/$B2C_STORE/browser');" projects/storefrontapp/server.ts
+then
+    echo "Dist folder has been updated"
+else
+    echo "Dist folder has NOT been updated"
+    exit 1
+fi
+
+echo "---------------------------------------------------------------------------------------------------------------------------"
+echo "Build SSR for b2c storefront"
 
-# to change
 yarn build:ssr:ci
 
-echo "-----"
-echo "Build CSR"
+echo "---------------------------------------------------------------------------------------------------------------------------"
+echo "Build CSR for b2c storefront"
 
 yarn build
 
-echo "-----"
-echo "Clone ccv2 repository"
+echo "---------------------------------------------------------------------------------------------------------------------------"
+echo "Copy server and browser files to js-storefront to adhere to the ccv2 dist structure for b2c storefront"
 
-git clone -b spa_p4_dist_test https://$GHT_USER:$GHT_PRIVATE_REPO_TOKEN@github.tools.sap/cx-commerce/$GHT_REPO.git
+cp -a dist/storefrontapp/. $CCV2_B2C_STOREFRONT_PATH/dist/$B2C_STORE/browser/
+cp -a dist/storefrontapp-server/. $CCV2_B2C_STOREFRONT_PATH/dist/$B2C_STORE/server/
 
-echo "-----"
-echo "Updating js-storefront to adhere to the ccv2 dist strucutre"
+echo "---------------------------------------------------------------------------------------------------------------------------"
+echo "update server.ts for b2b storefront"
 
-cd "$GHT_REPO/js-storefront"
-rm -rf $B2C_STORE
-rm -rf $B2B_STORE
+sed -i "s%dist/$B2C_STORE/browser%dist/$B2B_STORE/browser%gi" projects/storefrontapp/server.ts
 
-mkdir -p $B2C_STORE/dist/$B2C_STORE/browser
-mkdir -p $B2C_STORE/dist/$B2C_STORE/server
+echo "---------------------------------------------------------------------------------------------------------------------------"
+echo "Verify server.ts has been updated for b2b dist"
 
-# only b2c for now test
-# mkdir -p $B2B_STORE/dist/$B2B_STORE/browser
-# mkdir -p $B2B_STORE/dist/$B2B_STORE/server
+cat projects/storefrontapp/server.ts
 
-cd -
+if grep -Fq "const distFolder = join(process.cwd(), 'dist/$B2B_STORE/browser');" projects/storefrontapp/server.ts
+then
+    echo "Dist folder has been updated"
+else
+    echo "Dist folder has NOT been updated"
+    exit 1
+fi
+
+echo "---------------------------------------------------------------------------------------------------------------------------"
+echo "Build SSR for b2b storefront"
+
+export SPA_ENV='b2b'
```

Comment:
to help clarify why this is here is because in the package script, we allow to consume $SPA_ENV for any 'added/modified' env. We need to build for b2b so it was either using that mechanism or create a package script for b2b:build and buid:b2b:ssr etc